source/core/slang-string.h

```diff
@@ -510,6 +510,19 @@ namespace Slang
             return getData() + getLength();
         }
 
+        // Function to convert to std::string
+        std::string toStdString() const {
```

Comment:
We don't use std::string in this codebase. Use String.

Unless there is any specific reasons, I don't think the basic things like string need to be changed at this stage of the project.